functions/PayrollProcessor.Functions/Features/Departments/DepartmentsTrigger.cs

```diff
@@ -0,0 +1,129 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using PayrollProcessor.Functions.Features.Resources;
+using PayrollProcessor.Functions.Infrastructure;
+using System;
+using System.Threading.Tasks;
+using System.Linq;
+using PayrollProcessor.Functions.Features.Employees;
+using PayrollProcessor.Core.Domain.Features.Departments;
+using PayrollProcessor.Functions.Features.Employees.QueueMessages;
+
+namespace PayrollProcessor.Functions.Features.Departments
+{
+    public class DepartmentsTrigger
```

Comment:
I wonder how we should break up these triggers (or leave them together)? They feel like bulk controllers.